activity_browser/app/ui/tables/lca_results.py

```diff
@@ -29,4 +29,4 @@ def sync(self, lca):
                 self.setItem(row, col, ReadOnlyItem("{:.4g}".format(lca.results[row, col])))
 
         # ensure minimum height as it gets too small otherwise
-        self.setMinimumHeight(self.rowHeight(0) * self.rowCount() + self.autoScrollMargin())
+        self.setMinimumHeight(self.rowHeight(0) * (self.rowCount()) + self.autoScrollMargin())
```

Comment:
![image](https://user-images.githubusercontent.com/11636405/38765791-51cbc27e-3fc8-11e8-8585-ff77fecec17d.png)
Still very little space, especially for single activities. But I feel the score table needs to be reworked anyway, because not only the vertical space is problematic, but also the horizontal space for longer activity names. I suggest putting this in a separate issue instead of fixing it here. What do you think?

Copying the table with the index or even exporting to excel, csv is also something I've been meaning to implement for quite some time.

Reply:
Yes, the table got worse compared to the previous version. But the students should be working on it, so we will hopefully soon get an improvement.